shared/js/dataDriver.js

```diff
@@ -284,6 +301,7 @@ loop.DataDriver = function() {
       }));
       this._sse.addEventListener("open", this);
       this._sse.addEventListener("put", this);
+      this._sse.addEventListener("error", this);
```

Comment:
As discussed on IRC, one potential downside of not always refetching is that data can change while disconnected. File a separate bug to track this refetch or not issue.